src/app/api/shifts/route.ts

```
@@ -90,21 +89,44 @@ export async function GET(request: Request) {
-      liquorSales: Number(shift.liquorSales),
-      role: {
-        ...shift.role,
-        basePayRate: Number(shift.role.basePayRate),
-        configs: shift.role.configs.map(config => ({
-          ...config,
-          percentageRate: Number(config.percentageRate)
-        }))
+    const serializedShifts = shifts.map(shift => {
+      const shiftDate = new Date(shift.date);
+      let currentBasePayRate = 0;
+      if (shift.role && shift.role.configs && shift.role.configs.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)